src/cow-react/modules/swap/containers/EthFlowStepper/index.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  EthFlowStepper as Pure,
+  EthFlowStepperProps,
+  SmartOrderStatus,
+} from '@cow/modules/swap/pure/EthFlow/EthFlowStepper'
+import { useDetectNativeToken } from '@cow/modules/swap/hooks/useDetectNativeToken'
+import { Order } from 'state/orders/actions'
+import { NATIVE_CURRENCY_BUY_ADDRESS } from 'constants/index'
+
+type Props = {
```

Comment:
nit: why not calling it `EthFlowStepperProps` and  export it. This way cosmos or any other component can import it easily. I think we usually use that naming convention